package.json

```diff
@@ -116,11 +117,17 @@
       "default": true,
       "order": 5
     },
+    "enableDicy": {
+      "description": "Enable the usage of the [DiCy](https://yitzchak.github.io/dicy/) builder instead of `latexmk`.",
+      "type": "boolean",
```

Comment:
@thomasjo We could use an enumeration here and call the setting `builder` again?